Gemfile.6.1.pg.lock

```diff
@@ -42,9 +42,9 @@ GEM
     pry (0.14.2)
       coderay (~> 1.1)
       method_source (~> 1.0)
-    pry-byebug (3.10.1)
+    pry-byebug (3.8.0)
```

Comment:
Why the downgrade? :thinking:

Reply:
I assume this must have happened when I ran `bundle exec rake matrix:install` locally. The Gemfile is used for multiple ruby versions and maybe I ran a lower Version last, which required a lower version here. But then again, it's weird that it had been green in the CI test :man_shrugging: 

I just resetted it to the higher version